packages/cdk/resource/apiGatewayMetrics.ts

```diff
@@ -0,0 +1,40 @@
+import * as cw from "aws-cdk-lib/aws-cloudwatch"
+import {Duration} from "aws-cdk-lib"
+
+const apiGatewayNames = [
+  "pfp-apigw",
+  "psu-apigw",
+  "tracker-auth-apigw-cognito"
```

Comment:
dont need tracker-auth-apigw-cognito - this was just a demo one I set up
will need more soon as well though for fhir api, cpt, cpt-ui

Reply:
A metric for the `tracker-auth-apigw-cognito` has been removed.